examples/00-mapdl-examples/lathe_cutter.py

```diff
@@ -0,0 +1,422 @@
+"""
+.. _ref_lathe_cutter_example:
+
+=====================================
+Structural Analysis of a Lathe Cutter
+=====================================
+
+**Summary**: Basic walkthrough of PyMAPDL capabilities.
+
+Objective
+=========
+
+The objective of this example is to highlight some regularly used
+PyMAPDL features via a lathe cutter finite element model. Lathe
+cutters have multiple avenues of wear and failure, and the analyses
+supporting their design would most often be transient
+thermal-structural. However, for simplicity, this simulation
+example uses a non-uniform load.
+
+.. figure:: ../../../_static/lathe_cutter_model.png
+    :align: center
+    :width: 600
+    :alt:  Lathe cutter geometry and load description.
+    :figclass: align-center
+
+    **Figure 1: Lathe cutter geometry and load description.**
+
+
+
+Contents
+========
+
+1. **Variables and launch**
+   Define necessary variables and launch MAPDL.
+
+2. **Geometry, mesh, and MAPDL parameters**
+   Import geometry and inspect for MAPDL parameters. Define linear
+   elastic material model with Python variables. Mesh and apply symmetry
+   boundary conditions.
+
+3. **Coordinate system and load**
+   Create a local coordinate system for the applied load and verify with
+   a plot.
+
+4. **Pressure load**
+   Define the pressure load as a sine function of the length of the
+   application area using numpy arrays. Import the pressure array into
+   MAPDL as a table array. Verify the applied load and solve.
+
+5. **Plotting**
+   Show result plotting, plotting with selection, and working with the
+   plot legend.
+
+6. **Postprocessing:**
+   List a result two ways: use PyMAPDL and the Pythonic version of APDL.
+   Demonstrate extended methods and writing a list to a file.
+
+7. **Advanced plotting**
+   Use of `mesh.grid <https://mapdldocs.pyansys.com/mapdl_commands/graphics/_autosummary/ansys.mapdl.core.Mapdl.grid.html>`_
+   for additional postprocessing.
+
+
+Step 1: Variables and launch
+============================
+
+Define variables and launch MAPDL.
+"""
+
+import os
+
+import numpy as np
+
+from ansys.mapdl.core import launch_mapdl
+from ansys.mapdl.core.examples.downloads import download_example_data
+
+# cwd = current working directory
+path = os.getcwd()
```

Comment:
Is there any reason why we need to run MAPDL in the current working directory?

Reply:
Not on my side, but it was requirement of the course this example is taken from. I could remove it though.

Reply:
Please remove it. I think it's a bad practice.